java/dagger/example/android/espresso/main/AndroidManifest.xml

```diff
@@ -0,0 +1,20 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="com.example.android.espresso">
+
+  <uses-sdk
+    android:minSdkVersion="14"
+    android:targetSdkVersion="24"/>
+
+  <application
+    android:label="Espresso Dagger"
```

Comment:
Maybe just "Instrumentation Dagger"? 

Espresso is a concrete library for Instrumentation UI tests which not everybody use and not only it does not matter for this sample but it's also just an implementation detail.